plugins/repository-s3/src/main/java/org/opensearch/repositories/s3/S3Repository.java

```diff
@@ -171,6 +175,12 @@ class S3Repository extends MeteredBlobStoreRepository {
         }
     });
 
+    /**
+     * Type of Async client to be used for S3 Uploads.
+     * Defaults to crt.
+     */
+    static final Setting<String> S3_ASYNC_HTTP_CLIENT_TYPE = Setting.simpleString("s3_async_client_type", "crt");
+
```

Comment:
Assuming this will be the default going forward?

Reply:
Yes. Making this as default